backend/src/main/resources/db/migration/V168__update_booking_related_tbls.sql

```diff
@@ -0,0 +1,14 @@
+-- public.booking_links
+alter table public.booking_links add column location varchar(500);
+alter table public.booking_links add column note text;
+alter table public.booking_links add column removed boolean default false;
+drop index booking_links_owner_index;
+
+create index booking_links_removed_owner_index
+    on booking_links (removed, owner);
```

Comment:
It should be owner, removed

Reply:
Make the equality filter in the first place

Reply:
sure. updated.